generators/xml2jsonFile.ts

```diff
@@ -0,0 +1,72 @@
+import * as fs from "fs";
+import * as path from "path";
+import * as xml2js from "xml2js";
+
+type Config = {
+  icd10: {
+    xmlPath: string;
+  };
+};
+
+const parentPath = process.argv[2] || path.resolve(process.cwd(), "..", "..");
```

Comment:
You don't check whether there are at least 3 arguments

Reply:
Yes, I assume first argument added here will be path to `package.json`. We are not expected any other arguments for now, also I assume this won't be used by library users. If argument is not provided the value would be `undefined` and that is ok for me.